# Generated for NaoQi method ALAudioPlayer.playInt
# Starts the playback of a file preloaded with the loadFile function, with specific volume and audio balance. Asynchronous.

# Id returned by the loadFile function
int32 id
# volume of the sound file (must be between 0.0 and 1.0)
float32 volume
# audio balance of the sound file (-1.0 : left / 1.0 : right)
float32 pan
---
# Id of the file which has been loaded. This file can then be played with the play function
int32 value


